README.md

```diff
@@ -23,10 +34,21 @@ Production Installation
 
 Install the latest version of OMERO.server and OMERO.web and then:
 
-    cd $OMERO_HOME/lib/python/omeroweb
-    wget -O master.zip https://github.com/openmicroscopy/virtual-microscope/zipball/master
-    unzip master.zip
-    mv openmicroscopy-virtual-microscope-* virtual-microscope
-    path/to/bin/omero config set omero.web.apps '["virtual-microscope"]'
+    $ cd $OMERO_HOME/lib/python/omeroweb
+    $ wget -O master.zip https://github.com/openmicroscopy/virtual-microscope/zipball/master
+    $ unzip master.zip
+    $ mv openmicroscopy-virtual-microscope-* virtual-microscope
+
+Add virtual-microscope custom app to your installed web apps:
+
+    $ bin/omero config append omero.web.apps '"virtual-microscope"'
+
+NB: note that double quotes are wrapped by single quotes. Windows users will need to do
+
+    $ bin\omero config append omero.web.apps "\"virtual-microscope\""
+
+Redirect to your new post login page
+
+    $ bin/omero config set omero.web.login_redirect '{"redirect": ["webindex"], "viewname": "webindex_custom"}'
```

Comment:
Does that mean effectively that you can't do both? Down the road, is there possibility/need to refactor this to allow both since it certainly seemed confusing to at least me?